### PR TITLE
Closes Several Issues. 

### PR DESCRIPTION
**Pull request by axle2005**
Closes Issue #63 #66 #50 #54 #44

Commit f376de9 - Added a simple permission check to check if player had ```nucleus.connectionmessages.disable``` and return if true. 

Commit 336c4f9 - Added method setDescription to ChannelUtil and added to GameStoppingServerEvent event.

Commit bb73f49 - Added new variable, ```playerCount```. Increases and decreases as clients connect. Updates description using previous commit. 

Commit 063ac85 - Added a check for Discord messages starting with '!', using as an escape character to create discord commands. 

Commit d74003e - I screwed up making a package on the previous commit :P Shit happens lol 

Commit 0e9409f - Added 'removeColor' method to strip minecraft color codes from any message sent to discord. 

Commit 0b44d8f - Added Support for 'Boop'. Creates 'hooks' package for future support of plugins. 

Commit d743f3a - Moved the string for the Nucleus Staff channel to Nucleus class in hooks package created in previous commit.

### CODE REVIEW

**Comment by Mohron** [comment]
I'm going to borrow some of this for my fork if you don't mind :D

File: src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
  * Created by Hy on 1/4/2016.
  */
 @Plugin(id = "discordbridge", name = "Discord Bridge", version = "2.3.0",
-        description = "A Sponge plugin to connect your Minecraft server with Discord", authors = {"Hy", "Mohron"})
+        description = "A Sponge plugin to connect your Minecraft server with Discord", authors = {"Hy", "Mohron, Axle2005"})
```
Comment (Mohron) [inline src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java]: One PR makes you an Author? Interesting...

Comment (axle2005) [inline src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java]: Honestly :P If he wants to deny this one commit, I don't care. The rest of the PR is good, but I'm still going to be hobbling along on this plugin anyway.

Comment (Mohron) [inline src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java]: I just think it's funny when contributors add themselves.

Comment (axle2005) [inline src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java]: Time for another commit then: Instead of author, Grand Pumba of Randomness

Comment (Mohron) [inline src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java]: LMAO.

**Comment by Mohron** [comment]
Recommend splitting discord commands and Twitter support to their own PRs. Also if you would update the Readme and example configs with documentation of your additions.

**Comment by axle2005** [comment]
Well, Ill do some splitting and resubmit.

**Pull request closed without merging**